calico-cloud/image-assurance/scanners/registry-scanner.mdx

```diff
@@ -0,0 +1,147 @@
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
+
+To use this option, all you need to do is:
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI. Because the scanner is not running continuously, it may take some time for results to display in the dashboard, especially if you have many images or large images.
+
+## Before you begin
+
+**Required**
+
+- You must have valid registry credentials
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Google GCR 
+
+**Limitations**
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Setting up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a docker image. Simply run `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4` in order to retrieve it.
+
+### Setting up registry scanner
+
+#### Credentials
+
+The registry scanner supports scanning all registries as long as the docker credentials are provided.
+Providing the `DOCKER_USERNAME` and `DOCKER_PASSWORD` lets the registry scanner retrieve images that the credentials have access to. Retrieving images 
+using these credentials will only retrieve images with tags.
+
+If you have a valid /.docker/config.json, you can also mount these while running the registry scanner.
+```
+docker run -v ~/.docker/config.json:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
+```
+
+The registry scanner also supports authentication using AZURE and AWS credentials. If you use these credentials, the registry scanner is able to
+retrieve images with no tags associated with them.
+
+Azure credentials needed:
+`AZURE_CLIENT_ID` 
+`AZURE_CLIENT_SECRET` or `AZURE_FEDERATED_TOKEN`
+`AZURE_TENANT_ID`
+
+AWS credentials needed:
+`AWS_ACCESS_KEY_ID`
+`AWS_SECRET_ACCESS_KEY`
+`AWS_REGION`
+
+The registry scanner currently scans one registry at a time. If you specify the Azure or AWS credentials, the docker username and password will not be used.
+
+#### Required Inputs
+
+The registry scanner requires:
+ 
+`REGISTRY` - the registry you want to scan.
```

Comment:
Would be help to give an example value here or be more descriptive on what the value should be (e.g. it has to be the full URL of the registry).